stubs/six/six/__init__.pyi

```diff
@@ -63,9 +64,11 @@ def u(s: str) -> str: ...
 unichr = chr
 
 def int2byte(i: int) -> bytes: ...
-def byte2int(bs: bytes) -> int: ...
-def indexbytes(buf: bytes, i: int) -> int: ...
-def iterbytes(buf: bytes) -> _Iterator[int]: ...
+
+byte2int = operator.itemgetter(0)
```

Comment:
That's not legal in a stub.

Reply:
oops, didn't catch that. I accidentally ran the tests locally with `py` instead of `python` (venv fail)